### DISCUSSION TITLE
Azure Database for MySQL

### DISCUSSION THREAD

**Discussion by wadhahbenaissa** [tomkerkhove/promitor#1529] [Q&A] [answered]
hello,

I hope you are doing well!

I was wondering whether promitor resource discovery supports Azure DB forMySQL?

Kind Regards

**Comment by tomkerkhove** [comment, marked as answer]
You can find all supported scrapers on https://promitor.io/configuration/v2.x/metrics/#supported-azure-services

They are all convenience scrapers meaning that they are scoped to a single service, but if one is not supported you can still rely on the generic scraper - https://promitor.io/configuration/v2.x/metrics/generic-azure-resource

So it is supported (through generic) but a dedicated one is being tracked in https://github.com/tomkerkhove/promitor/issues/324

**Reply by wadhahbenaissa** [reply]
Thank you @tomkerkhove for your quick interaction.
Can you please explain to me the usage of "filter" (The filter to use to have fine-grained metrics)

**Reply by tomkerkhove** [reply]
Filter is optional, in some cases you'd like to filter based on a dimension of the metric.

**Reply by wadhahbenaissa** [reply]
thanks a lot @tomkerkhove for your help, now I get it.

**Reply by tomkerkhove** [reply]
Happy to help!
